### PR TITLE
BAEL-3212 - Examples of POJO, Bean and BeanUtils reflection

### PR DESCRIPTION
**Pull request by JonathanTurnock**
@ashleyfrieze - Pull Request for the associated draft article.

### CODE REVIEW

File: spring-core-2/src/main/java/com/baeldung/pojo/EmployeePojo.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        this.startDate = startDate;
+    }
+
+    String name() {
+        return this.firstName + " " + this.lastName;
+    }
+
+    LocalDate getStart() {
+        return this.startDate;
+    }
```
Comment (ashleyfrieze) [inline spring-core-2/src/main/java/com/baeldung/pojo/EmployeePojo.java]: These methods should be `public` rather than `package-private`

Comment (ashleyfrieze) [reply spring-core-2/src/main/java/com/baeldung/pojo/EmployeePojo.java]: Or is the point that they're not `public` so it's not a bean?

Comment (JonathanTurnock) [reply spring-core-2/src/main/java/com/baeldung/pojo/EmployeePojo.java]: @ashleyfrieze I have corrected these to be public now.

File: core-java-modules/core-java-lang-2/src/main/java/com/baeldung/pojo/EmployeeBean.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+public class EmployeeBean {
+
+    private String firstName;
+
+    private String lastName;
+
+    private LocalDate startDate;
+
+    public EmployeeBean() {
+
```
Comment (ashleyfrieze) [inline core-java-modules/core-java-lang-2/src/main/java/com/baeldung/pojo/EmployeeBean.java]: `MINOR` - we don't need the blank line here

File: core-java-modules/core-java-lang-2/src/main/java/com/baeldung/pojo/EmployeeBean.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
+package com.baeldung.pojo;
+
+import java.time.LocalDate;
+
+public class EmployeeBean {
```
Comment (ashleyfrieze) [inline core-java-modules/core-java-lang-2/src/main/java/com/baeldung/pojo/EmployeeBean.java]: In the article, you say that beans must be Serializable... this isn't - or at least it doesn't implement `Serializable`

**Pull request merged**